include/LightGBM/config.h

```diff
@@ -218,6 +218,12 @@ struct Config {
   // desc = this seed has lower priority in comparison with other seeds, which means that it will be overridden, if you set other seeds explicitly
   int seed = 0;
 
+  // desc = used only with ``cpu`` device type
```

Comment:
Can we have a fatal error in `config.cpp` in case of `true` and non-CPU device for those who don't love to read docs? 🙂